include/CDash/Database.php

```diff
@@ -15,7 +15,6 @@
 =========================================================================*/
 namespace CDash;
 
-use CDash\Log;
```

Comment:
this turned out to be unnecessary?  Are we not using Log inside Database?

Reply:
This is still using `add_log` so, currently, CDash\Log is unused.